samples/package.json

```diff
@@ -4,12 +4,14 @@
   "license": "Apache-2.0",
   "author": "Google Inc.",
   "repository": "googleapis/nodejs-translate",
+  "files": [
```

Comment:
this is what kicks out all the test files from the config, and I guess sets a good example for developers :)